extensions/src/test/java/org/pentaho/platform/plugin/services/email/EmailConfigurationTest.java

```diff
@@ -0,0 +1,36 @@
+/*!
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ *
+ * Copyright (c) 2002-2023 Hitachi Vantara. All rights reserved.
+ *
+ */
+
+package org.pentaho.platform.plugin.services.email;
+
+import junit.framework.TestCase;
+
+public class EmailConfigurationTest extends TestCase {
+
+  public void testEqualsEmailConfig() {
```

Comment:
need test for EmailConfiguration#equals when two objects are no equal. verifying that equals actually has logic that compares all properties. 

You can get away by with just changing one property of two "identical" objects.